database/factories/VenueFactory.ts

```diff
@@ -7,7 +7,7 @@ export default Factory.define(Venue, ({ faker }) => {
     description: faker.random.words(),
     street: faker.address.street(),
     streetNumber: faker.random.numeric(),
```

Comment:
Interessant. postgres oder adonis wandelt das in einen string um weil es im Model ein string ist